.github/workflows/build-complete-samples.yml

```diff
@@ -843,6 +843,10 @@ jobs:
               name: 'tab-navbar-menu'
               version: '16.x'
 
+            - project_path: 'samples/bot-types-of-cards/nodejs'
+              name: 'bot-types-of-cards'
+              version: '18.x'
```

Comment:
Kindly run build pipeline and add the screenshot.

Reply:
Executed Build Pipeline and added ss at first comment of this PR.